Makefile

```diff
@@ -0,0 +1,64 @@
+JOBS:=$(shell nproc)
+
+.PHONY: clean build-developer build-developer-debug \
+	build-byoc build-byoc-debug build-factory \
+	generate-cli-parsers run-edge-core all \
+	run-edge-core-resetting-storage build-doc
+
+clean:
+	rm -rf build build-doc build-nondebug config/edge_version_info.h
+
+build:
+	mkdir -p build
+
+generate-cli-parsers:
+	cd edge-core && ./gen_docopt.sh
+
+lib/mbed-cloud-client/source/update_default_resources.c: lib/mbed-cloud-client
+	manifest-tool init -d "mbed.com" -m "mbed-edge" -q
```

Comment:
This probably need some changes for manifest-tool 2.0 version.

Reply:
Yes, this should be changed to `manifest-dev-tool`. Last time when I did that, the PR checks started to fail. I guess as we have not yet enabled the same tests in this repo's PR checks, its fine to make this change and fix what breaks later.

Reply:
Done.